### PR TITLE
KB-4909 - added completed tab under myiGot strip 

### PR DESCRIPTION
**Pull request by garima468**
KB-4909 - added completed tab under myiGot strip

### CODE REVIEW

File: library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.html [hunk 1]
```diff
@@ -1592,9 +1592,9 @@ <h2 class="mat-subheading-2 flex-1 min-w-0 margin-remove-bottom" i18n>{{'cardcon
 
 <ng-template #cbpPlan  let-data="data">
   <div class="cbp-mark left-0 border-top-radius">
-    <p *ngIf="data.cbpData.planDuration === acbpConstants?.UPCOMING"  class="cbp-warning">{{data.cbpData.endDate | date: 'd MMM, y'}}</p>
-    <p *ngIf="data.cbpData.planDuration === acbpConstants?.SUCCESS"  class="cbp-success">{{data.cbpData.endDate | date: 'd MMM, y'}}</p>
-    <p *ngIf="data.cbpData.planDuration === acbpConstants?.OVERDUE" class="cbp-danger">{{'cardcontentv2.overDue' | translate}}</p>
+    <p *ngIf="data.cbpData.planDuration === acbpConstants?.UPCOMING && data?.cbpData?.contentStatus < 2"  class="cbp-warning">{{data.cbpData.endDate | date: 'd MMM, y'}}</p>
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.html]: do optional check

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts [hunk 2]
```diff
@@ -1167,17 +1167,20 @@ export class ContentStripWithTabsComponent extends WidgetBaseComponent
...
   getSelectedIndex(stripsResultDataMap: any, key: any): number {
     let returnValue = 0
     if (key === 'cbpPlan') {
       if (stripsResultDataMap.tabs.length) {
-        const data = stripsResultDataMap.tabs.filter((ele: any) => ele.value === 'upcoming')
-        returnValue = data[0].widgets && data[0].widgets.length > 0 ? 1 : 0
+        // const data = stripsResultDataMap.tabs.filter((ele: any) => ele.value === 'upcoming')
+        // returnValue = data[0].widgets && data[0].widgets.length > 0 ? 1 : 0
+        const data = stripsResultDataMap.tabs.filter((ele: any) => ele.value === 'upcoming' || ele.value === 'overdue')
+        returnValue = (data[0].widgets && data[0].widgets.length > 0) ? 0 : (data[1].widgets && data[1].widgets.length > 0) ? 1 : 2
```
Comment (vishnubansaltarento) [inline library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts]: please check if data available then use data[0]

File: src/app/cbp/cbp-plan/cbp-plan.component.ts [hunk 3]
```diff
@@ -97,17 +99,17 @@ export class CbpPlanComponent implements OnInit {
           this.upcommingList.push(ele)
         }
       })
-
+      this.completedList = response.filter((allData: any) => allData.contentStatus === 2)
```
Comment (vishnubansaltarento) [inline src/app/cbp/cbp-plan/cbp-plan.component.ts]: please use 2 from constant

File: src/app/component/cbp-plan-stats/cbp-plan-stats.component.html [hunk 4]
```diff
@@ -23,6 +18,16 @@
             <ws-widget-skeleton-loader *ngIf="cbpLoader" [width]="'28px'" [height]="'20px'" [bindingClass]="'flex rounded h-16 margin-right-xs'"></ws-widget-skeleton-loader>
             <p class="group-count" *ngIf="!cbpLoader">{{cbpCount?.overdue || 0}}</p>
         </div>
+        <div>
+            <p class="group-title">{{'cbpPlanStats.completed' | translate }}</p>
```
Comment (vishnubansaltarento) [inline src/app/component/cbp-plan-stats/cbp-plan-stats.component.html:22]: please do optional check

File: library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.html (hunk 1 above)
Comment (garima468) [reply library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.html]: resolved this.

File: library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts (hunk 2 above)
Comment (garima468) [reply library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts]: done this changes

File: src/app/cbp/cbp-plan/cbp-plan.component.ts (hunk 3 above)
Comment (garima468) [reply src/app/cbp/cbp-plan/cbp-plan.component.ts]: done this change

File: src/app/component/cbp-plan-stats/cbp-plan-stats.component.html (hunk 4 above)
Comment (garima468) [reply src/app/component/cbp-plan-stats/cbp-plan-stats.component.html:22]: if we are making optional for translate text, it is breaking on UI

**Pull request merged**